### PR TITLE
add eslint airbnb

### PR DESCRIPTION
**Pull request by arjun0419**
https://trello.com/c/5SaZ4x6U/13-xs-configure-eslint

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  "extends": "airbnb",
+  "globals" : {
+    "document": true,
+
```
Comment (lee-philip-n) [inline .eslintrc.js]: extra space here is your eslint working?

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  "extends": "airbnb",
+  "globals" : {
```
Comment (lee-philip-n) [inline .eslintrc.js]: env:
    browser: true

**Comment by lee-philip-n** [comment]
added two comments, redo this before I accept the merge

**Comment by arjun0419** [comment]
@lee-philip-n  incorporated your comments. Please re-review.

**Pull request merged**